pilot/pkg/bootstrap/server_test.go

```diff
@@ -203,7 +287,7 @@ func TestReloadIstiodCert(t *testing.T) {
 	}
 
 	// setup cert watches.
-	if err := s.initCertificateWatches(tlsOptions); err != nil {
```

Comment:
I think passing tlsOptions is not a bad idea ( instead of N args ). More stable API.

Reply:
Understood , will amend this, as well as the path location as commented above